satellite/gracefulexit/endpoint.go

```diff
@@ -345,58 +293,22 @@ func (endpoint *Endpoint) doProcess(stream processStream) (err error) {
 		if !morePiecesFlag && pendingCount == 0 {
 			processMu.Unlock()
 
-			exitStatusRequest := &overlay.ExitStatusRequest{
-				NodeID:         nodeID,
-				ExitFinishedAt: time.Now().UTC(),
-			}
-
 			progress, err := endpoint.db.GetProgress(ctx, nodeID)
```

Comment:
I think this `GetProgress` and the monkit tracking can also be moved into `generateExitStatusRequest`